### PR TITLE
Document recommended java opts for running tests

### PR DESCRIPTION
**Pull request by sunahsuh**
As mentioned by @maurodoglio in IRC

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-batch-view/131)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: README.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ If you run into memory issues during compilation time issue the following comman
 export _JAVA_OPTIONS="-Xss4M -Xmx2G"
 ```
 
+The test suite is memory intensive and run best with these java opts:
+```bash
+export _JAVA_OPTIONS="-Xms4G -Xmx4G -Xss4M -XX:MaxMetaspaceSize=256M"
```
Comment (vitillo) [inline README.md]: Let's merge this with the previous paragraph.

**Comment by sunahsuh** [comment]
Okay, updated the existing paragraph

**Comment by vitillo** [comment]
Thanks!

**Pull request merged**